### ISSUE TITLE
Is the "scheme" property supported?

### ISSUE THREAD

**Issue by Denny09310** [christianhelle/apiclientcodegen#1055]
**Describe the bug**
I'm trying to generate using Refitter the client fot [this](https://github.com/user-attachments/files/18079824/openapi.json) openapi json file, but even when using the option useIsoDateFormat the [Query] attribute is not generated using the Format property. After trying to debug the problem I found out that in this [line of code](https://github.com/christianhelle/refitter/blob/a8fd0dbbcfb3d4c7f11e4b2b824e7259d31e1540/src/Refitter.Core/ParameterExtractor.cs#L93) the property `IsDateOrDateTime` is `false`. I couldn't investigate further unfortunately, because I didn't found the root cause

**Support Key**: `6fb4645` 

**To Reproduce**
1. Go to '[UseIsoDateFormatTests.cs](https://github.com/christianhelle/refitter/blob/a8fd0dbbcfb3d4c7f11e4b2b824e7259d31e1540/src/Refitter.Tests/Examples/UseIsoDateFormatTests.cs)'
2. Change the `'format'` and `'type'` properties of one of the dates to be inside the `'schema'` object (like in the generated file above)
3. Try to run the test
4. See the test fail

**Expected behavior**
Scaffold the client correctly using the `'schema'` property

**Visual Studio Extension (please complete the following information):**
 - Visual Studio 2022
 - Extension Version 1.17.0

**Comment by christianhelle** [comment]
@Denny09310 Thanks for taking the time to report this. I'll see what I can do using the OpenAPI specifications you attached.

This issue might be better to be reported in Refitter, but since I'm also the author of Refitter, it doesn't really matter that much 😄

**Comment by Denny09310** [comment]
Yes, sorry, I wasn't sure if I should have posted it here or in the Refitter repository. Since I'm using it through the Visual Studio extension I decided to give as much information as possible 😅
